js/researches/getPassiveVoice.js

```diff
@@ -1,326 +1,57 @@
 var getSentences = require( "../stringProcessing/getSentences.js" );
-var arrayToRegex = require( "../stringProcessing/createRegexFromArray.js" );
-var stripSpaces = require( "../stringProcessing/stripSpaces.js" );
 var stripHTMLTags = require( "../stringProcessing/stripHTMLTags.js" ).stripFullTags;
-var matchWordInSentence = require( "../stringProcessing/matchWordInSentence.js" );
-var normalizeSingleQuotes = require( "../stringProcessing/quotes.js" ).normalizeSingle;
+var getLanguage = require( "../helpers/getLanguage.js" );
+var Sentence = require( "../values/Sentence.js" );
 
-var nonverbEndingEd = require( "./english/passivevoice-english/non-verb-ending-ed.js" )();
-var determiners = require( "./english/passivevoice-english/determiners.js" )();
+// English.
+var getSentencePartsEnglish = require( "./english/getSentenceParts.js" );
```

Comment:
It is nice to have a language config like file where this will be organized like 

```js
var config = {
    en : {
       sentenceParts:  require( "./english/getSentenceParts.js" ),
       passives: require( "./english/determinePassives.js" ),
    },

    de : {
       sentenceParts:  require( "./german/getSentenceParts.js" ),
       passives: require( "./german/determinePassives.js" )
    }
};
```